templates/docs/examples/patterns/navigation/subnav-dark.html

```diff
@@ -15,7 +15,7 @@
       <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
       <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
     </div>
-    <nav class="p-navigation__nav">
+    <nav  aria-label="Example main navigation">
```

Comment:
You removed the class name by mistake

```suggestion
    <nav class="p-navigation__nav" aria-label="Example main navigation">
```